docker-compose.test.yml

```diff
@@ -21,6 +21,7 @@ services:
       - FHIR_ARCHIVE_DIR=../bcdaworker/archive
       - OKTA_CLIENT_ORGURL=https://cms-sandbox.oktapreview.com
       - OKTA_EMAIL=shawn@bcda.aco-group.us
+      - BCDA_FHIR_MAX_RECORDS=100
```

Comment:
I didn't see this env var being used by the tests. Is this needed?

Reply:
I wasn't sure. I would rather add it now than be super confused about what is happening later when it isn't there.